go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	k8s.io/api v0.23.0
 	k8s.io/apimachinery v0.23.0
 	k8s.io/client-go v0.23.0
+	k8s.io/code-generator v0.23.0 // indirect
```

Comment:
This requires Scott's security review...

Reply:
this line was removed - it maybe added during dev. It is not needed after using `go mod tidy`